x-pack/plugins/alerting/server/task_runner/transform_action_params.ts

```diff
@@ -89,3 +98,48 @@ export function transformActionParams({
     variables
   );
 }
+
+export function transformSummaryActionParams({
+  alerts,
+  rule,
+  ruleTypeId,
+  actionsPlugin,
+  actionId,
+  actionTypeId,
+  spaceId,
+  actionParams,
+  ruleUrl,
+  kibanaBaseUrl,
+}: {
+  alerts: SummarizedAlertsWithAll;
+  rule: SanitizedRule<RuleTypeParams>;
+  ruleTypeId: string;
+  actionsPlugin: ActionsPluginStartContract;
+  actionId: string;
+  actionTypeId: string;
+  spaceId: string;
+  actionParams: RuleActionParams;
+  kibanaBaseUrl?: string;
+  ruleUrl?: string;
+}): RuleActionParams {
+  const variables = {
+    kibanaBaseUrl,
+    date: new Date().toISOString(),
+    rule: {
+      params: rule.params,
```

Comment:
**optional nit:** would be cool if we also add `rule.params` within `transformActionParams` above so that the `rule` object is the same for both.

Reply:
Done in [#3d137a1](https://github.com/elastic/kibana/pull/147360/commits/3d137a13893d66c7b29ac93e20446f4fefbba12c)